---
title: "Robust phenotype prediction from gene expression data using differential shrinkage of co-regulated genes"
collection: publications
permalink: /publications/Robust phenotype prediction from gene expression data using differential shrinkage of co-regulated genes
venue: "Scientific Reports"
date: 19 January 2018
citation: 'K. Zarringhalam, <b>D. Degras</b>, C. Brockel, D. Ziemek, 8(1), 1237 - January 2018'
---

### Abstract
Discovery of robust diagnostic or prognostic biomarkers is a key to optimizing therapeutic benefit for select patient cohorts - an idea commonly referred to as precision medicine. Most discovery studies to derive such markers from high-dimensional transcriptomics datasets are weakly powered with sample sizes in the tens of patients. Therefore, highly regularized statistical approaches are essential to making generalizable predictions. At the same time, prior knowledge-driven approaches have been successfully applied to the manual interpretation of high-dimensional transcriptomics datasets. In this work, we assess the impact of combining two orthogonal approaches for the discovery of biomarker signatures, namely (1) well-known lasso-based regression approaches and its more recent derivative, the group lasso, and (2) the discovery of significant upstream regulators in literature-derived biological networks. Our method integrates both approaches in a weighted group-lasso model and differentially weights gene sets based on inferred active regulatory mechanism. Using nested cross-validation as well as independent clinical datasets, we demonstrate that our approach leads to increased accuracy and generalizable results. We implement our approach in a computationally efficient, user-friendly R package called [creNET](https://github.com/kouroshz/creNethttps://github.com/kouroshz/creNet) and is accompanied by a parsed version of the STRING DB data base.


[Download paper here](https://www.nature.com/articles/s41598-018-19635-0)
